### ISSUE TITLE
Can't use locked levers.

### ISSUE THREAD

**Issue by andrewkm** [Tsuser1/Modern-LWC#95]
I've added levers to our core config.
`        lever:
            enabled: true`
You can successfully lock them, however you can no longer use them.
Even the owner of the lock cannot use the levers.

Setup is all latest as of this post.
Paper: 1.13.2 Build 571
ModernLWC built on: https://github.com/Tsuser1/Modern-LWC/commit/288b1bff7e60825b95a2070185e82f4297027032

**Comment by andrewkm** [comment]
Looks like a workaround is to use /credstone on - on the lever after locking it.
Not sure if this is the plugins intended functionality, but thought I'd mention that.

**Comment by Tsuser1** [comment]
While this issue doesn't functionally break the usability of levers in-game, I do still see the issue with this confusing system and people not being aware of `/credstone`. Due to the flexible and powerful nature of MLWC (and LWC in general), many people aren't aware of the _full_ scope of options available.

My opinion is this is acting as intended, but perhaps a message when locking redstone objects to inform players of this could be beneficial.

**Comment by andrewkm** [comment]
A message would be great :)

**Comment by pop4959** [comment]
@andrewkm Sorry for the long delayed response, did `/credstone` fix the issue for you? If so, I think we can close this issue.

**Comment by pop4959** [comment]
I have tested this and confirmed that it is working as intended. You probably have denyRedstone set to true, which by default will block redstone on protections. If you don't want it to do this, set it to false.

From the config (protections.denyRedstone):
If true, the default behaviour will be to BLOCK redstone to protections, and /credstone on (Protection Flag)
If false, the default behaviour will be to ALLOW redstone to protections, and /credstone on will BLOCK redstone to the protection

Closing issue now since this is not a bug.
